korean/eating-walls/prepare-hot-choco.md

```diff
@@ -0,0 +1,7 @@
+핫초코를 끓이다 보니 마시멜로 벽이 녹아 없어지고 있어요!
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.